pkg/telemetryservice/utils/k8s.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+	"os"
+)
+
+var clientSet *kubernetes.Clientset
+var ns string
+
+func initClient() error {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return err
+	}
+	clientSet, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+	ns = os.Getenv("EDGEDEVICE_NAMESPACE")
+	return nil
+}
+
+func GetPasswordFromSecret(name string) (string, error) {
```

Comment:
we should also get the username, filling in username and password in two different places seems inconvenient